project/Dependencies.scala

```diff
@@ -58,4 +58,8 @@ object Dependencies {
   lazy val schematron = Seq(
     "net.sf.saxon" % "Saxon-HE" % "11.2",
   )
+
+  lazy val log4jcore = Seq(
```

Comment:
I'd suggest calling this `tdml_processor`. If we ever add new `tdml_processor dependencies`, they can go in this variable.